db/schema.rb

```diff
@@ -11,14 +11,25 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20180815155404) do
+ActiveRecord::Schema.define(version: 20180821190945) do
 
   # These are extensions that must be enabled in order to support this database
   enable_extension "plpgsql"
   enable_extension "uuid-ossp"
   enable_extension "pg_trgm"
   enable_extension "btree_gin"
 
+  create_table "accounts", force: :cascade do |t|
+    t.uuid     "uuid",       null: false
+    t.string   "idme_uuid"
+    t.string   "icn"
+    t.string   "edipi"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+  end
+
+  add_index "accounts", ["uuid"], name: "index_accounts_on_uuid", unique: true, using: :btree
```

Comment:
is that unique on the idme_uuid missing?

Reply:
No, this is for the index associated with the new `account#uuid` method being unique.

For the `idme_uuid` attr, we are enforcing uniqueness at both the database layer, and the model layer.  And validating for presence at the model layer.  

Not sure if I answered exactly what you were asking?